dist/docker/redhat/Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV container docker
 
 # The SCYLLA_REPO_URL argument specifies the URL to the RPM repository this Docker image uses to install Scylla. The default value is the Scylla's unstable RPM repository, which contains the daily build.
 ARG SCYLLA_REPO_URL=http://downloads.scylladb.com/rpm/unstable/centos/master/latest/scylla.repo
-
+ARG VERSION
```

Comment:
Good idea. But please add a default (666) so it also works without arguments.

Reply:
@avikivity It idea was to make sure the docker will always get the right Scylla version that match the relevant centos-rpm build. We can put a default, but then, we can have a situation that we have a new docker image with older Scylla version inside.

Reply:
I want to preserve the ability to do a "docker build -f dist/docker/redhat .", not everyone can go through jenkins.

Reply:
I agree, @yaronkaikov can you please update the pull request?

Reply:
Yes. so default value will be `666.development`?

Reply:
Changed and tested locally